pygeos/test/test_linear.py

```diff
@@ -81,5 +81,4 @@ def test_shared_paths_none():
 def test_shared_paths_non_linestring():
     g1 = pygeos.linestrings([(0, 0), (1, 0), (1, 1)])
     g2 = pygeos.points(0, 1)
-    with pytest.raises(pygeos.GEOSException):
-        pygeos.shared_paths(g1, g2)
+    raise Exception(pygeos.shared_paths(g1, g2))
```

Comment:
Why the change here? (it looks a bit strange)

Reply:
I must have wanted to see the result of `pygeos.shared_paths(g1, g2)` in the console. It should go out